### PR TITLE
apparmor: various fixes

### PR DESCRIPTION
**Pull request by paper42**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

required by aa-notify

### CODE REVIEW

**Comment by ericonr** [comment]
I will hold onto this for a bit while I try to figure out the linux 5.4 issue, so we can push a single time.

**Comment by paper42** [comment]
then I will add some more improvements into this PR

* add missing dependency python3-notify2 for aa-notify
* do not rewrite logfiles option in logprof.conf aggressively
* remove an old patch

File: srcpkgs/apparmor/template [hunk 1]
```diff
@@ -34,7 +34,7 @@ pre_build() {
 	cp ${FILESDIR}/profiles/* profiles/apparmor.d/
 
 	# use the correct syslog path
-	vsed -i utils/logprof.conf -e 's,logfiles = .*,logfiles = /var/log/socklog/kernel/current,'
+	vsed -i utils/logprof.conf -e 's,logfiles = .*,logfiles = /var/log/audit/audit.log /var/log/socklog/kernel/current /var/log/syslog /var/log/messages,'
```
Comment (ericonr) [inline srcpkgs/apparmor/template:37]: Please split the expression into its own line.

**Comment by ericonr** [comment]
Just pull this patch (thankfully someone noticed the issue, because I wasn't sure how I would solve it) https://gitlab.com/apparmor/apparmor/-/commit/cc113f4820721808c9efec8b075a5482e6f9a3ad

And put in the commit/PR description that it fixes #28127

Thanks

File: srcpkgs/apparmor/template (hunk 1 above)
Comment (paper42) [reply srcpkgs/apparmor/template]: moved to the logprof patch

File: srcpkgs/apparmor/template [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+revision=2
 wrksrc="${pkgname}-v${version}"
 build_wrksrc=libraries/libapparmor
 build_style=gnu-configure
 conf_files="/etc/apparmor.d/local/* /etc/apparmor/*"
 make_dirs="/etc/apparmor.d/disable 0755 root root"
 hostmakedepends="bison flex autoconf automake libtool gettext swig python3 which"
 makedepends="perl python3-devel"
-depends="runit-void-apparmor python3 libapparmor"
+depends="runit-void-apparmor python3 libapparmor python3-notify2"
```
Comment (ericonr) [inline srcpkgs/apparmor/template:12]: Turns out you also need `python3-psutils` here.

**Pull request closed without merging**